src/app/routes/utils/regex/utils/index.js

```diff
@@ -32,11 +32,54 @@ export const getArticleManifestRegex = services => {
   return `/:service(${serviceRegex})/:local(${articleLocalRegex})/manifest.json`;
 };
 
+const frontPageServicesToMigrate = [
+  'kyrgyz',
```

Comment:
Can we move kyrgyz to the appropriate location in this list, so that it is in alphabetical order? (This will also require an update to the src/app/routes/utils/regex/utils/__snapshots__/index.test.js.snap file by running the failing test file with a `-u` at the end)